### PR TITLE
docs(react-swatch-picker): added description and best practices

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
## New Behavior
- Returned back BestPractices because we don't have it yet on Fluent 2 website
- Added description to all the stories
- Changed order of stories
- Made image example smaller
- Removed images from Layout stories
- Added "MixedSwatches" story
- Made "Shape" story using the same layout everywhere

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Best practices
```
Comment (layershifter) [inline packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md:1]: Let's wrap this with `<details>` & `<summary>` like for other components.

**Review by layershifter — APPROVED** [review]

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md (hunk 1 above)
Comment (ValentinaKozlova) [reply packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md]: fixed

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### Do
+
+- Maximum recommended number of swatches per row is 8.
+- Use a grid layout when there are more than 4 swatches. The maximum recommended number of swatches in a grid is 64 - an 8 x 8 grid.
+- Tooltip should be shown for each color.
+
+### Don't
+
+- Avoid using rounded corners in a grid layout for the SwatchPicker, as it may cause the Hermann grid illusion.
```
Comment (marcosmoura) [inline packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerBestPractices.md:14]: Can we add a link that references to an explanation of this Hermann Grid Illusion definition?

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerLayout.stories.tsx [hunk 3]
```diff
@@ -20,47 +19,24 @@ const useStyles = makeStyles({
   },
 });
 
-const colors: ColorSwatchProps[] = [
+const colors = [
   { color: '#FF1921', value: 'FF1921', 'aria-label': 'red' },
   { color: '#FF7A00', value: '#FF7A00', 'aria-label': 'orange' },
```
Comment (marcosmoura) [inline packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerLayout.stories.tsx:24]: Is the value different here on purpose?

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerMixedSwatches.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    ...shorthands.margin('20px', '0'),
+    '@media (forced-colors: active)': {
+      forcedColorAdjust: 'none',
+    },
+  },
+});
+
+const colors: ColorSwatchProps[] = [
+  { color: '#FF1921', value: 'FF1921', 'aria-label': 'red' },
+  { color: '#FF7A00', value: '#FF7A00', 'aria-label': 'orange' },
```
Comment (marcosmoura) [inline packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerMixedSwatches.stories.tsx:24]: Same here

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerSize.stories.tsx [hunk 5]
```diff
@@ -52,3 +52,11 @@ export const SwatchPickerSize = () => {
     </div>
   );
 };
+
+SwatchPickerSize.parameters = {
+  docs: {
+    description: {
+      story: 'The `size` prop sets width and height of the Swatch. The default is `medium`.',
```
Comment (marcosmoura) [inline packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerSize.stories.tsx:59]: Is there any reference somewhere to the differences between the sizes? If not, can that be added here?

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerLayout.stories.tsx (hunk 3 above)
Comment (ValentinaKozlova) [reply packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerLayout.stories.tsx:24]: Actually, it should be without "#"

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerSize.stories.tsx (hunk 5 above)
Comment (ValentinaKozlova) [reply packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerSize.stories.tsx:59]: What differences do you mean?

File: packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerMixedSwatches.stories.tsx (hunk 4 above)
Comment (ValentinaKozlova) [reply packages/react-components/react-swatch-picker-preview/stories/SwatchPicker/SwatchPickerMixedSwatches.stories.tsx:24]: fixed

**Pull request merged**